huaweicloud/provider.go

```diff
@@ -791,7 +791,7 @@ func Provider() *schema.Provider {
 			"huaweicloud_iotda_device_certificate":  iotda.ResourceDeviceCertificate(),
 			"huaweicloud_iotda_device_linkage_rule": iotda.ResourceDeviceLinkageRule(),
 
-			"huaweicloud_kms_key":     ResourceKmsKeyV1(),
+			"huaweicloud_kms_key":     dew.ResourceKmsKeyV1(),
```

Comment:
the function name should be without version suffix

Reply:
ok,done